src/Runner.Listener/JobDispatcher.cs

```diff
@@ -1129,34 +1146,12 @@ private async Task LogWorkerProcessUnhandledException(IRunnerService server, Pip
                     TimelineRecord jobRecord = timeline.Records.FirstOrDefault(x => x.Id == message.JobId && x.RecordType == "Job");
                     ArgUtil.NotNull(jobRecord, nameof(jobRecord));
 
-                    try
-                    {
-                        if (!string.IsNullOrEmpty(errorMessage) &&
-                            message.Variables.TryGetValue("DistributedTask.EnableRunnerIPCDebug", out var enableRunnerIPCDebug) &&
```

Comment:
do you mind deleting the entire code for `DistributedTask.EnableRunnerIPCDebug` together? I don't think we need those anymore since it was added for debugging a customer's issue and that's resolved now.

Reply:
done!